app/build.gradle

```diff
@@ -1,35 +1,47 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    signingConfigs {
+        config {
+            storeFile file('/Applications/Android Studio.app/Contents/bin/mykey')
```

Comment:
Please don't hard-code platform / OS specific paths in here (let alone macOS paths 😁 ).